avro-fastserde/src/codegen/java/com/linkedin/avro/fastserde/generated/deserialization/AVRO_1_4/StringableRecord_SpecificDeserializer_6174384286732341990_6174384286732341990.java

```diff
@@ -87,7 +87,7 @@ public StringableRecord_SpecificDeserializer_6174384286732341990_617438428673234
                 chunkLen0 = (decoder.arrayNext());
             } while (chunkLen0 > 0);
         } else {
-            urlArray0 = Collections.emptyList();
+            urlArray0 = new ArrayList<Utf8>();
```

Comment:
nit - initial size 1?

Reply:
why not 0 since it's always an empty ArrayList? :)

Reply:
I don't really like the structure of that generated code, and I was meaning to tweak it anyway at some point... so might as well do it now.